catalog/ui/catalog-ui-search/src/main/webapp/component/input/textarea/input-textarea.hbs

```diff
@@ -0,0 +1,21 @@
+{{!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ --}}
+<div class="if-editing">
+    <textarea id="{{cid}}" value="{{value}}" placeholder="{{property.placeholder}}" name="{{property.id}}"></textarea>
```

Comment:
Looks fine to me (at least on the side).  It's possible their mail service could be susceptible to an xss attack though, so it might not hurt to sanitize it for their sake.